lib/open_food_network/reports/variant_overrides.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module OpenFoodNetwork
+  module Reports
+    class VariantOverrides
+      def initialize(line_items:, distributor_ids:)
```

Comment:
This class should not take line_items, they are not needed, you can just take variant_ids and call with line_items.select(:variant_id)

Reply:
Done.